kong.yml

```diff
@@ -94,10 +94,19 @@
 ######
 ## PostgreSQL configuration
 # postgres:
-#    host: "127.0.0.1"
-#    port: 5432
-#    user: kong
-#    database: kong
+  #####
+  ## Connection host and port settings
+  # host: "127.0.0.1"
+  # port: 5432
+
+  ######
+  ## Name of the database used by Kong. Will be created if it does not exist.
+  # database: kong
+
+  #####
+  ## User authentication settings
+  # user: ""
+  # password: ""
```

Comment:
The main reason why Cassandra has detailed explanations is because generally, people are less familiar with its particular properties (replication factor, strategy, consistency, etc...). There is no point in writing comments (code-wise, and in this configuration file) if the comment simply state what the thing (code of property) already does in appearance. A comment needs to explain either **why**, or **how**, or reveal an non-apparent behavior. For example: ` Will be created if it does not exist.`, or `Provide a user and a password here if your cluster uses the PasswordAuthenticator scheme.`.

For example, a relevant comment for Postgres' `user` and `password` would be to specify that the supported authentication schemes are Trust and MD5 only ([source](https://github.com/leafo/pgmoon#authentication-types)).

In any way, the new configuration file will have new comments and different property names so this is only relevant for 0.8.0's lifetime.